src/LiveComponent/composer.json

```diff
@@ -38,6 +38,7 @@
         "symfony/dependency-injection": "^5.4|^6.0",
         "symfony/form": "^5.4|^6.0",
         "symfony/framework-bundle": "^5.4|^6.0",
+        "symfony/maker-bundle": "^1.48",
```

Comment:
Seems to fix an issue with Foundry, where its `make:factory` command was exploding without this.

Reply:
```suggestion
```

The issue in foundry has been fixed

Reply:
![bam](https://media.giphy.com/media/ywkTx4yhxqweTO46mT/giphy.gif)